client/src/components/Pages/CausativeEvent/CausativeEvent.tsx

```diff
@@ -9,6 +9,7 @@ import {
   RadioGroup,
   TextField,
 } from "@material-ui/core";
+import React from "react";
```

Comment:
idk why these imports weren't there already or how these files were working/not crashing without these imports...